### ISSUE TITLE
About pcre

### ISSUE THREAD

**Issue by bleakwind** [krakjoe/pthreads#596]
I Can't find my post.

When I doing this, pcre.h problem is fix.

`# ln -s /usr/local/include/pcre.h /usr/local/include/php/ext/pcre/pcre.h`

But there is still some error.

`# make
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c -o src/socket.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c  -fPIC -DPIC -o src/.libs/socket.o
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:123:21: warning: implicit declaration of function 'if_nametoindex' is invalid in C99 [-Wimplicit-function-declaration]
        unsigned int ind = if_nametoindex(val);
                           ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:469:6: warning: implicit declaration of function 'php_set_sock_blocking' is invalid in C99 [-Wimplicit-function-declaration]
        if (php_set_sock_blocking(threaded->store.sock->fd, blocking) != SUCCESS) {
            ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:562:3: error: non-void function 'pthreads_sockets_to_fd_set' should return a value [-Wreturn-type]
                PTHREADS_SOCKET_CHECK(threaded->store.sock);
                ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:33:3: note: expanded from macro 'PTHREADS_SOCKET_CHECK'
                return; \
                ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:601:10: warning: implicit declaration of function 'add_assoc_str_zval' is invalid in C99 [-Wimplicit-function-declaration]
                                if (!add_assoc_str_zval(&set, key, element)) {
                                     ^
3 warnings and 1 error generated.
*** Error code 1

Stop.
make: stopped in /pub/_bleakwind/php-7.0.6/ext/pthread`

**Comment by BurakDev** [comment]
Your first issue is here #595

**Comment by bleakwind** [comment]
It wors, thanks. But still have some warning. If u can, maybe fix this.

> creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
root@server:/pub/_bleakwind/php-7.0.6/ext/pthread # make
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/php_pthreads.c -o php_pthreads.lo
mkdir .libs
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/php_pthreads.c  -fPIC -DPIC -o .libs/php_pthreads.o
In file included from /pub/_bleakwind/php-7.0.6/ext/pthread/php_pthreads.c:30:
./src/object.h:42:1: warning: '/*' within block comment [-Wcomment]
/* {{{ */
^
1 warning generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/monitor.c -o src/monitor.lo
mkdir src/.libs
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/monitor.c  -fPIC -DPIC -o src/.libs/monitor.o
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/stack.c -o src/stack.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/stack.c  -fPIC -DPIC -o src/.libs/stack.o
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/globals.c -o src/globals.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/globals.c  -fPIC -DPIC -o src/.libs/globals.o
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/prepare.c -o src/prepare.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/prepare.c  -fPIC -DPIC -o src/.libs/prepare.o
In file included from /pub/_bleakwind/php-7.0.6/ext/pthread/src/prepare.c:26:
./src/object.h:42:1: warning: '/*' within block comment [-Wcomment]
/* {{{ */
^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/prepare.c:652:22: warning: '&' within '|' [-Wbitwise-op-parentheses]
        if (thread->options & PTHREADS_INHERIT_CLASSES|PTHREADS_INHERIT_FUNCTIONS) {
            ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~
/pub/_bleakwind/php-7.0.6/ext/pthread/src/prepare.c:652:22: note: place parentheses around the '&' expression to silence this
      warning
        if (thread->options & PTHREADS_INHERIT_CLASSES|PTHREADS_INHERIT_FUNCTIONS) {
            ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/store.c -o src/store.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/store.c  -fPIC -DPIC -o src/.libs/store.o
In file included from /pub/_bleakwind/php-7.0.6/ext/pthread/src/store.c:30:
./src/object.h:42:1: warning: '/*' within block comment [-Wcomment]
/* {{{ */
^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/store.c:348:36: warning: '&&' within '||' [-Wlogical-op-parentheses]
                if (IS_PTHREADS_OBJECT(write) || IS_PTHREADS_CLOSURE(write)) {
                                              ~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~
./src/pthreads.h:103:27: note: expanded from macro 'IS_PTHREADS_CLOSURE'
        Z_TYPE_P(z) == IS_OBJECT && instanceof_function(Z_OBJCE_P(z), zend_ce_closure)
        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/pub/_bleakwind/php-7.0.6/ext/pthread/src/store.c:348:36: note: place parentheses around the '&&' expression to silence this
      warning
                if (IS_PTHREADS_OBJECT(write) || IS_PTHREADS_CLOSURE(write)) {
                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
./src/pthreads.h:103:27: note: expanded from macro 'IS_PTHREADS_CLOSURE'
        Z_TYPE_P(z) == IS_OBJECT && instanceof_function(Z_OBJCE_P(z), zend_ce_closure)
        ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/resources.c -o src/resources.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/resources.c  -fPIC -DPIC -o src/.libs/resources.o
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/handlers.c -o src/handlers.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/handlers.c  -fPIC -DPIC -o src/.libs/handlers.o
In file included from /pub/_bleakwind/php-7.0.6/ext/pthread/src/handlers.c:26:
./src/object.h:42:1: warning: '/*' within block comment [-Wcomment]
/* {{{ */
^
1 warning generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/object.c -o src/object.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/object.c  -fPIC -DPIC -o src/.libs/object.o
In file included from /pub/_bleakwind/php-7.0.6/ext/pthread/src/object.c:26:
./src/object.h:42:1: warning: '/*' within block comment [-Wcomment]
/* {{{ */
^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/object.c:197:3: warning: passing 'unsigned char *' to parameter of type 'char *'
      converts between pointers to integer types with different sign [-Wpointer-sign]
                (*buffer), ":%lu:", (long unsigned int) address);
                ^~~~~~~~~
/usr/local/include/php/main/snprintf.h:87:31: note: passing argument to parameter 's' here
PHPAPI int php_sprintf (char* s, const char* format, ...) PHP_ATTRIBUTE_FORMAT(printf, 2, 3);
                              ^
2 warnings generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=compile cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c -o src/socket.lo
 cc -DZEND_ENABLE_STATIC_TSRMLS_CACHE=1 -I. -I/pub/_bleakwind/php-7.0.6/ext/pthread -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/. -DHAVE_CONFIG_H -g -O2 -c /pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c  -fPIC -DPIC -o src/.libs/socket.o
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:131:21: warning: implicit declaration of function 'if_nametoindex' is invalid
      in C99 [-Wimplicit-function-declaration]
        unsigned int ind = if_nametoindex(val);
                           ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:477:6: warning: implicit declaration of function 'php_set_sock_blocking' is
      invalid in C99 [-Wimplicit-function-declaration]
        if (php_set_sock_blocking(threaded->store.sock->fd, blocking) != SUCCESS) {
            ^
/pub/_bleakwind/php-7.0.6/ext/pthread/src/socket.c:609:10: warning: implicit declaration of function 'add_assoc_str_zval' is
      invalid in C99 [-Wimplicit-function-declaration]
                                if (!add_assoc_str_zval(&set, key, element)) {
                                     ^
3 warnings generated.
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=link cc -DPHP_ATOM_INC -I/pub/_bleakwind/php-7.0.6/ext/pthread/include -I/pub/_bleakwind/php-7.0.6/ext/pthread/main -I/pub/_bleakwind/php-7.0.6/ext/pthread -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -I/pub/_bleakwind/php-7.0.6/ext/pthread/.  -DHAVE_CONFIG_H  -g -O2   -o ./pthreads.la -export-dynamic -avoid-version -prefer-pic -module -rpath /pub/_bleakwind/php-7.0.6/ext/pthread/modules  php_pthreads.lo src/monitor.lo src/stack.lo src/globals.lo src/prepare.lo src/store.lo src/resources.lo src/handlers.lo src/object.lo src/socket.lo 
cc -shared  .libs/php_pthreads.o src/.libs/monitor.o src/.libs/stack.o src/.libs/globals.o src/.libs/prepare.o src/.libs/store.o src/.libs/resources.o src/.libs/handlers.o src/.libs/object.o src/.libs/socket.o   -Wl,-soname -Wl,pthreads.so -o ./.libs/pthreads.so
creating ./pthreads.la
(cd ./.libs && rm -f pthreads.la && ln -s ../pthreads.la pthreads.la)
/bin/sh /pub/_bleakwind/php-7.0.6/ext/pthread/libtool --mode=install cp ./pthreads.la /pub/_bleakwind/php-7.0.6/ext/pthread/modules
cp ./.libs/pthreads.so /pub/_bleakwind/php-7.0.6/ext/pthread/modules/pthreads.so
cp ./.libs/pthreads.lai /pub/_bleakwind/php-7.0.6/ext/pthread/modules/pthreads.la
